contracts/prize-strategy/multiple-winners/MultipleWinners.sol

```diff
@@ -37,12 +37,12 @@ contract MultipleWinners is PeriodicPrizeStrategy {
   /**
     * @dev Emitted when a new MultipleWinnersPrizeSplit config is added.
   */
-  event PrizeSplitSet(address indexed target, uint16 percentage, TokenType token);
+  event PrizeSplitSet(address indexed target, uint16 percentage, TokenType token, uint8 index);
 
   /**
     * @dev Emitted when a MultipleWinnersPrizeSplit config is removed.
   */
-  event PrizeSplitRemoved(address indexed target);
+  event PrizeSplitRemoved(address indexed target, uint8 index);
```

Comment:
I don't think we need to emit the `target`, as it's in the `Set` event.

also- we should make the index `indexed`